### ISSUE TITLE
Configure Cognito verification email custom sender address

### ISSUE THREAD

**Issue by mir1198yusuf** [aws-amplify/amplify-cli#6985]
I have setted up auth using amplify cli. When users signs up, they receive verification code email from no-reply@verification email address.

I want to change that and use my gmail address. I tried searching on GitHub issues and on internet how to do that using amplify cli. The only method I could find was using AWS Cognito Userpool console. I heard that any manual changes made to console gets lost after next amplify push operation.

Will console method work or any other method using amplify cli is possible?

Thanks!

**Comment by siegerts** [comment]
Hi @mir1198yusuf , thanks for the question. The above method to configure is the **correct** approach currently - using Cognito console + Verifying the email as an SES identity.

These changes will persist across `amplify push` unless the [Auth] resource is destroyed and then recreated.  Also, in that situation, the email can be reused, if needed, once it's verified in SES.

- https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses.html
- https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses-procedure.html

**Comment by mir1198yusuf** [comment]
Thanks for the answer!

**Comment by mir1198yusuf** [comment]
I could not set up custom sender email for default Cognito configuration.

My cognito user pool region : ap-south-1

My userpool Message customization section's SES Region sub-section shows these region
![image](https://user-images.githubusercontent.com/34382235/113296240-d5c5b680-9316-11eb-8dd2-8faa2048208b.png)

I went to SES and added two new email addresses for ap-south-1 region and US East Virginia (1st one in above image) regions.

Now in my userpool Message customization section's From Email address ARN sub-section , I can see two entries in dropdown - Default and other my email address added in US East Virginia (1st one in SES region , not the one added in ap-south-1 which is userpool region)


Below i kept it default :
![image](https://user-images.githubusercontent.com/34382235/113296639-5ab0d000-9317-11eb-9fcf-9c090a56ce3f.png)

Now in my userpool Message customization section's From Email address (not Email address ARN) sub-section : 

![image](https://user-images.githubusercontent.com/34382235/113296843-99df2100-9317-11eb-927f-63f8f1189e56.png)

Email address ARN - default
Email address - blank

When saved, still getting verification email from no-reply@verification


Email address ARN - one added in SES US East Viriginia
Email address - blank

When saved, still getting verification email from no-reply@verification



Email address ARN - default OR one added in SES (both selected one-by-one)
Email address - one added in SES

When saved,  getting error
![image](https://user-images.githubusercontent.com/34382235/113297229-fd694e80-9317-11eb-9e49-6e94e082addc.png)


Basically I want to use default Cognito config (50 emails per day) but custom sender email which I am not getting to work 

Please help!

I have used aws amplify cli to add auth by amplify add auth command

**Comment by siegerts** [comment]
H @mir1198yusuf - apologies that this is giving you some trouble.

In the top portion - Select the region and then enter then enter the email that is verified via SES.
![image](https://user-images.githubusercontent.com/634958/113427777-a1e4a280-93a3-11eb-8fb4-477b98df5561.png)

Then, select the **Yes, use Amazon SES** option

![image](https://user-images.githubusercontent.com/634958/113428004-069ffd00-93a4-11eb-99b8-448d2df18a9f.png)

**Comment by mir1198yusuf** [comment]
Hello @siegerts  -
I followed the above steps and it really helped.
But the issue I am facing now is that after signing, I am receiving following error -

> Cognito received the following error from Amazon SES when attempting to send email : Email address is not verified. The following identities failed the check in region US-EAST-1 : <new_signed_up_email>

My Cognito userpool is of ap-south-1 and SES verified email (custom sender email for Cognito) is of us-east-1

Do I have to move SES out of sandbox? or any other

**Comment by siegerts** [comment]
Hi @mir1198yusuf , Yes, you'll need to verify the email and move it out of the sandbox.  The full details of this process are documented @ https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-email.html .

Also, from the docs - 

> Available regions for Amazon Cognito are us-east-1, us-east-2, us-west-2, eu-west-1, eu-west-2, eu-central-1, ap-northeast-1, ap-northeast-2, ap-south-1, ap-southeast-1, ap-southeast-2, and ca-central-1. Available Amazon SES regions are: us-east-1, us-west-2, eu-west-1.

Hope that this helps!

**Comment by mir1198yusuf** [comment]
After moving out of sandbox for region us-east-1, it is working perfectly. Thanks!

Sidenote - Add this to Amplify Auth doc please

**Comment by mir1198yusuf** [comment]
@siegerts -
Sorry to bother you again for unrelated thing. just one last thing, the verification code emails are going to Promotion folder.
I know this is not the right place to ask for this.
Can you redirect me where to file an issue on GitHub/etc ?

Any additional configs needed ?

Security details of email is 
Signed by - amazonses.com
